src/dev/flang/fuir/Clazz.java

```diff
@@ -608,14 +505,14 @@ public Set<Clazz> heirs()
    */
   private Set<Clazz> directParents()
   {
-    var result = new TreeSet<Clazz>();
+    var result = new HashSet<Clazz>();
```

Comment:
Why Hashsets now? As far as I can see the compareTo code did not change?

Reply:
There was some bug in compare, but that is fixed, all `HashSet`s should be remove.d